libs/coin-modules/coin-mina/src/test/api.unit.test.ts

```diff
@@ -0,0 +1,45 @@
+import { setConfig } from "./testUtils";
+setConfig();
+import { getAccount, getTransactions } from "../api";
+
+test("get balance for an account", async () => {
+  const invalidAddress = [
+    "B62qkdFWJSW8zaTBZjTVtmeU3rVxyUkNxPhKKW8T2JBtpj5XfdywLJM",
+    "novalidaddress",
+  ];
+  const validAddress = ["B62qkdFWJSW8zaTBZjTVtmeU3rVxyUkNxPhKKW8T2JBtpj5XfdywLSM"];
+
+  for (const address of invalidAddress) {
+    await expect(getAccount(address)).toBeDefined();
+  }
+
+  for (const address of validAddress) {
+    const account = await getAccount(address);
```

Comment:
If you call a backend here, you need to create a jest.integ.config.js file and run it with test-integ command (see package.json of other integrations)

You can't use backend in a .uni.test.ts (or test.ts aswell)

Reply:
Done, made testing better in general.